pilot/model/validation.go

```diff
@@ -1679,7 +1679,7 @@ func ValidateEndUserAuthenticationPolicySpec(msg proto.Message) error {
 		if jwt.JwksUri == "" {
 			errs = multierror.Append(errs, errors.New("jwks_uri must be set"))
 		}
-		if !strings.HasPrefix(jwt.JwksUri, "http") || !strings.HasPrefix(jwt.JwksUri, "https") {
+		if !strings.HasPrefix(jwt.JwksUri, "http") && !strings.HasPrefix(jwt.JwksUri, "https") {
```

Comment:
`"http://"` and `"https://"`? If the former condition met the latter one met automatically if they are "http" and "https"